modules/persistent-storage-snapshots-creating-deleting.adoc

```diff
@@ -0,0 +1,124 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent_storage/persistent-storage-snapshots.adoc
+
+[id="persistent-storage-snapshots-using_{context}"]
+= Creating and deleting snapshots
+
+Similar to how a persistent volume claim (PVC) binds to a persistent volume (PV) to provision a volume, `VolumeSnapshotData` and `VolumeSnapshot` are used to create a volume snapshot.
+// k8s docs refer to `VolumeSnapshotContent` instead of `VolumeSnapshotData`. Need to confirm which is correct here.
+
+Volume snapshots must use a supported `PersistentVolume` type.
+
+== Create snapshot
+
+To take a snapshot of a PV, create a new `VolumeSnapshotData` object based on the `VolumeSnapshot`, as shown in the following example:
+
+[source,yaml]
+----
+apiVersion: volumesnapshot.external-storage.k8s.io/v1
+kind: VolumeSnapshot <1>
+metadata:
+  name: snapshot-demo
+spec:
+  persistentVolumeClaimName: ebs-pvc <2>
+----
+<1> A `VolumeSnapshotData` object is automatically created based on the
+`VolumeSnapshot`.
+
+<2> `persistentVolumeClaimName` is the name of the `PersistentVolumeClaim` bound to a `PersistentVolume`. This particular PV is snapshotted.
+
+Depending on the PV type, the `Create Snapshot` operation might go through several phases, which are reflected by the `VolumeSnapshot` status:
```

Comment:
There's no need to put `Create Snapshot` in literals. This is the operation, but it's not a literal string or key. Remove the backticks and make this lowercase.